src/qutip_qip/device/circuitqed.py

```diff
@@ -69,7 +69,7 @@ def __init__(
                 )[: self.num_qubits]
             ),
             "wr": self.to_array(5.96, num_qubits - 1),
-            "alpha": self.to_array(-1, num_qubits),
+            "alpha": self.to_array(-0.3, num_qubits),
```

Comment:
Why -0.3? Is this documented?

Reply:
It is a commonly used parameter in the experiment. The anharmonicity is between 0.25~0.35GHz, e.g. in the two papers that the dosc refers to. I changed it to `-1` for debugging but forget to change it back.